app/controllers/pet_applications_controller.rb

```diff
@@ -27,11 +26,17 @@ def create
     end
   end
 
+  def update
+    pet = Pet.find(params[:pet_id])
+    pet.update(adopt_status: "Pending")
+    flash[:approve_application] = "The application has been approved!"
```

Comment:
We have 3 types of flash messages made, :error, :success, and :notices.  If you tag :approve_application with one of those it will auto format.